### PR TITLE
Create rawfetch.py

### PR DESCRIPTION
**Pull request by emilylxf**

### CODE REVIEW

File: rawfetch.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# This code can fetch data from the website, but it needs to login and get cookies handly. 
+# Because the cookies changes when login at different time, so new cookies and data should be use, 
+# if you want to make this code work.   
+# More work need to be done to make the fetch work automotically.
+from bs4 import BeautifulSoup
+import requests
+import csv
+
+#fetch data from the website
```
Comment (kis87988) [inline rawfetch.py:9]: this is under function def.
```py
def fn():
   '''
   doc string here
   '''
```

File: rawfetch.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#how to get the cookies and data?1-login the website, 2-click right mouse-inspect-network-click refresh- click the first of the left 
+#column-choose headers-then all the information are here
+def fetchdata():
+    url = "https://ssb-prod.ec.fhda.edu/PROD/bwskfcls.P_GetCrse_Advanced"
+    cookies = dict(SESSID='TlJCV1JJMTkzMjQwMw==', AWSELB='8FC3B10C7C6F2E81FB85778EC82E80B35F6053EB3FB0A326C415E62DF224C6255601225080A4895B1FADB06E83D852FA6F81C67EE98881214B71E413F60117E73BEF0DB1',shib_idp_session='929644161fc2ac3b1694fdb45c1f9b89343665d7655bd8d380d505660fe18c2b',IDMSESSID='608050D4F1FB8791B3DE6BA6CE1CD30D6BB7E87CC6D28205E263CFDD18FB7C89B87904542F7037A0BAC33AF2D4787F522A0A9D73A8D1CF8754DED595C67FF92C8BE78596A938A2D17308051D07FAB282')
```
Comment (kis87988) [inline rawfetch.py:14]: good job, but this should be save into one of `tmp` folder

File: rawfetch.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    r = requests.post(url=url, data=data, cookies=cookies)
+    #print(r.content)
+    print(r.text)
```
Comment (kis87988) [inline rawfetch.py:18]: try to use `loger`, but first PR use print is fine

File: rawfetch.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    return (r.text)
+  
+# save the data to a list
```
Comment (kis87988) [inline rawfetch.py:21]: same here. docs should below function def.

File: rawfetch.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    r = requests.post(url=url, data=data, cookies=cookies)
+    #print(r.content)
+    print(r.text)
+    return (r.text)
+  
+# save the data to a list  
+def get_contents(ulist, text):
+    soup = BeautifulSoup(text, 'lxml')
+    trs = soup.find_all('tr')
```
Comment (kis87988) [inline rawfetch.py:24]: maybe `table_row` is meaningful variable name

**Review by kis87988 — CHANGES_REQUESTED** [review]

File: rawfetch.py (hunk 1 above)
Comment (emilylxf) [reply rawfetch.py:9]: OK! Thank you very much!

File: rawfetch.py (hunk 2 above)
Comment (emilylxf) [reply rawfetch.py:14]: What do you mean that I should save it into one of tmp folder?

File: rawfetch.py (hunk 3 above)
Comment (emilylxf) [reply rawfetch.py:18]: Do you mean that I don't need to print always, but here is fine?

File: rawfetch.py (hunk 4 above)
Comment (emilylxf) [reply rawfetch.py:21]: OK! Thanks a lot!

**Pull request closed without merging**